2nd-gen/packages/swc/components/badge/stories/badge.stories.ts

```diff
@@ -76,74 +79,101 @@ export default meta;
 type BadgeVariant = typeof Badge.prototype.variant;
 type BadgeSize = typeof Badge.prototype.size;
 type FixedValues = typeof Badge.prototype.fixed;
-/**
- * `<sp-badge>` elements display a small amount of color-categorized metadata. They're ideal for getting a user's attention.
- */
+
 export const Playground: Story = {
     args: {
         size: 'm',
     },
     tags: ['autodocs', 'dev'],
 };
 
-// ─────────────────────
-//    USAGE STORIES
-// ─────────────────────
+// ──────────────────────────
+//    ANATOMY STORIES
+// ──────────────────────────
 
 /**
- * Badges can be rendered with or without an icon. Icons can be passed to the component using the `icon` slot and can be sourced from either the Spectrum icon library or a custom icon library as needed.
+ * A badge is made up of the following parts:
+ *
+ * - Text can be displayed within the badge by using the default slot
+ * - An optional icon element can be used to display an icon within the badge
+ *
+ * Badges can contain either a label, an icon, or both.
  */
-export const WithIcon: Story = {
+export const Anatomy: Story = {
+    render: (args) => html`
+        ${template({ ...args, 'default-slot': 'Label only' })}
+        ${template({ ...args, 'icon-slot': '✓', 'default-slot': '' })}
+        ${template({
+            ...args,
+            'icon-slot': '✓',
+            'default-slot': 'Icon and label',
+        })}
+    `,
+    tags: ['anatomy'],
     args: {
-        ['icon-slot']: '✓',
+        size: 'm',
     },
-    // Removes the story from the side navigation while keeping in the docs view
-    tags: ['usage'],
 };
 
+// ──────────────────────────
+//    OPTIONS STORIES
+// ──────────────────────────
+
 /**
- * Semantic variants allow you to render the badge with a descriptive name that maps to a design-system-aligned color. This is the preferred way to assign color to a badge because it will align more consistently with other components in your UI with the same meaning.
+ * Badges come in four sizes to fit various contexts:
  */
-export const SemanticVariants: Story = {
+export const Sizes: Story = {
```

Comment:
The array mapping is showing up in the docs code snippet instead of the rendered HTML, so what if we try this?

```suggestion
const sizesTemplate = CONTAINER(Badge.VALID_SIZES.map(
                (size) => html`
                    <swc-badge size=${size as BadgeSize}
                        >${capitalize(size)}</swc-badge
                    >
                `
            ));
export const Sizes: Story = {
```